v22.2/set-locality.md

```diff
@@ -1,296 +0,0 @@
----
```

Comment:
It looks like this file is empty and not fully deleted. Did you mean to delete this file?

Reply:
Yes! Thanks for the catch.